configs/defs/generic_linux.textproto

```diff
@@ -9558,3 +9558,467 @@ benchmark_configs: {
       "}"
   }
 }
+benchmark_configs: {
+  id: "ensure-libpam-modules-installed"
+  compliance_note: {
+    title: "Ensure libpam-modules is installed"
+    description:
+      "Pluggable Authentication Modules for PAM. Ensures the system has full functionality and access to the PAM options "
+      "covered by this Benchmark."
+    rationale:
+      "To ensure the system has full functionality and access to the PAM options covered by this Benchmark."
+    remediation:
+      "If the version of libpam-modules on the system is less than version 1.5.2-6, run the following command to update to "
+      "the latest version:\n"
+      "```\n"
+      "apt update && apt upgrade libpam-modules\n"
+      "```\n"
+    cis_benchmark: {
+      profile_level: 1
+      severity: MEDIUM
+    }
+    scan_instructions:
+      "generic:{check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/var/lib/dpkg/status\"}}"
```

Comment:
The "ensure X is installed" checks seem specific to debian-based systems. Are these benchmarks from the Ubuntu doc? Maybe we should move them to the Ubuntu config file then.